core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadWriteLDAPUserStoreManager.java

```diff
@@ -1378,8 +1381,7 @@ public void doUpdateRoleListOfUser(String userName, String[] deletedRoles, Strin
                                         searchBase);
                             } else {
                                 errorMessage =
-                                        "User: " + userName + " does not belongs to role: " +
-                                                deletedRole;
+                                        "User: " + userName + " does not belongs to role: " + deletedRole;
```

Comment:
Are we no displaying the username and deleted role with the dialog ? if so we need to encode this too right ?

Reply:
Username cannot be injected from UI at this point. However, added the encoding in 062e8994dd16aab6122fc02ebc3d871099030c16